website/source/docs/builders/virtualbox-vm.html.md.erb

```diff
@@ -239,6 +239,13 @@ builder.
     doesn't shut down in this time, it is an error. By default, the timeout is
     `5m` or five minutes.
 
+-   `disable_shutdown` (boolean) - Packer normally halts the virtual machine after all provisioners have
+    run when no `shutdown_command` is defined.  If this is set to `true`, Packer
+    *will not* halt the virtual machine but will assume that you will send the stop
+    signal yourself through the preseed.cfg or your final provisioner.
+    Packer will wait for a default of 5 minutes until the virtual machine is shutdown.
+    The timeout can be changed using `shutdown_timeout` option.
+
```

Comment:
Ah, intersting this virtualbox-vm.html.md.erb file does not have a 

```ruby
<%= partial "partials/builder/virtualbox/common/ShutdownConfig-not-required" %>
```

Like the virtualbox-ovf or virtualbox-iso files !

Reply:
I found this interesting as well 🤔 I thought I was doing something wrong.

Reply:
Maybe we can update that in another PR so it could be consistent with the other 🤔

Reply:
Nope that's not you ! I think this file was added while the docs refactor was being done and it was forgotten.

If you want you can also change the code this part something like this: 

https://github.com/hashicorp/packer/blob/8ac97de8b248397c355f8449569c016e14a752cb/website/source/docs/builders/virtualbox-ovf.html.md.erb#L114-L118